prov/usnic/src/usnic_direct/usd_poll.c

```diff
@@ -213,6 +213,27 @@ usd_poll_cq_multi(
 }
 
 int
+usd_check_empty(
```

Comment:
mentioned offline, but just so we don't lose track:

Let's move this functionality up to libfabric and leave usnic_direct alone for this.  Also, let's add `cq` to the name somewhere.